### DISCUSSION TITLE
FSR parameters

### DISCUSSION THREAD

**Discussion by motoleon** [Blinue/Magpie#43] [General]
How can i configure FSR parameters (quality, performance, etc)?
I am testing the utility with games.
Many thanks for it.

**Comment by Blinue** [comment]
FSR itself does not have these parameters, you can only adjust its sharpness. [Here](https://github.com/Blinue/Magpie/wiki/%E8%87%AA%E5%AE%9A%E4%B9%89%E7%BC%A9%E6%94%BE) is how to change the sharpness, but it is in Chinese.

**Comment by motoleon** [comment]
I only see in ScaleModels.json :

{
    "name": "FSR",
    "model": [
      {
        "module": "FFX",
        "effect": "FSR",
        "scale": [-1, -1]
      }
    ]
  },

where is sharpness parameter?

Thanks.

**Reply by Blinue** [reply]
try this:
{
"name": "FSR",
"model": [
{
"module": "FFX",
"effect": "FSR",
"scale": [-1, -1],
"sharpness": 1
}
]
}

**Reply by motoleon** [reply]
Ha, ok. Many Thanks.

**Reply by motoleon** [reply]
I will try it.

**Reply by FinFetChannel** [reply]
As I understand, from the FSR documentation, the presets only relate to the ratio between input and output resolution, so you have to set this manually. But thanks for the sharpness tip, will test it now  =)

**Reply by virtualbs** [reply]
If you need more sharpness, add a CAS step afterwards, for example:

```
  {
    "name": "FSR",
    "model": [
      {
        "module": "FFX",
        "effect": "FSR",
        "scale": [-1, -1],
        "sharpness": 1.0
      },
      {
        "module": "FFX",
        "effect": "CAS",
        "sharpness": 0.4
      }
    ]
  },
```
